lib/Zonemaster/CLI.pm

```diff
@@ -669,7 +669,7 @@ sub to_idn {
         return Zonemaster::LDNS::to_idn( decode( $self->encoding, $str ) );
     }
     else {
-        say STDERR __( "Warning: Zonemaster::LDNS not compiled with libidn, cannot handle non-ASCII names correctly." );
+        say STDERR __( "Warning: Zonemaster::LDNS not compiled with libidn2, cannot handle non-ASCII names correctly." );
```

Comment:
I think the following would be better and not bound to details not relevant here.
```
Warning: Zonemaster::LDNS not compiled with IDN support, cannot handle names with non-ASCII characters correctly.
```
But I accept as-is.

Reply:
Good point. I'll do the change in another PR.